src/shared/components/palette/dynamic-radio-list/dynamic-radio-list.pipe.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { DynamicRadioListPipe } from './dynamic-radio-list.pipe';
+import { FixedListItem } from '../../../domain/definition/fixed-list-item.model';
+
+describe('FixedRadioListPipe', () => {
+
+  const FIXED_LIST: FixedListItem[] = [
+    {
+      code: 'M',
+      label: 'Male',
+      order: 1
+    },
+    {
+      code: 'F',
+      label: 'Female',
+      order: 2
+    },
+    {
+      code: 'O',
+      label: 'Other',
+      order: 3
+    }
+  ];
+  const EMPTY = '';
+
+  let fixedRadioListPipe: DynamicRadioListPipe;
```

Comment:
_Nitpick:_
This could simply be instantiated here (as a `const`) rather in a `beforeEach(...)` block. 🤷‍♂️